internal/controller/config/rbac/agentserviceconfig_editor_role.yaml

```diff
@@ -0,0 +1,24 @@
+# permissions for end users to edit agentserviceconfigs.
```

Comment:
not sure if we need these roles.

Reply:
Wouldn't we at least need this one to assign to our operator? Or are we not creating a rolebinding also?